README.md

```diff
@@ -185,6 +185,10 @@ Add the following github secrets for the CD pipeline to work automatically:
 - MONGO_CONNECTION_STRING = Your Mongo Connection String
 - DBNAME = Mongo DB Name
 - COLLECTIONAME = Mongo Collection Name
+- WEBHOOK_URL = Azure Webhook URL to pull the image from Docker Hub and redeploy/restart
+
+## Azure Deployment 
+App is deployed to Azure ['https://laks-devops-todoapp.azurewebsites.net'](https://laks-devops-todoapp.azurewebsites.net)
```

Comment:
The app looks like it's deployed correctly but does have a bug - you're reusing the OAuth client you set up for Heroku so the Azure app redirects to GitHub but GitHub is redirecting back to your Heroku application! (Meaning that hitting that URL results in me reaching <http://todoapp-devops.herokuapp.com/>)